.github/pr-comments/for-writer.md

```diff
@@ -0,0 +1,10 @@
+:smiley_cat: Thank you for creating this PR! To publish your content to [Zowe Docs](https://docs.zowe.org/), follow these steps.
+
+- [ ] Add the label `review: sme` if an SME needs to check your content.
+- [ ] Identify your content topic with a label. (Examples: `area: apiml`, `area: cli`, `area: install and config`, etc.)
+- [ ] Specify the major Zowe release(s) for your content. (Examples: `release: V1`, `release: V2`, `release: V3`)
+    - [ ] If adding content that needs to be removed from V3 documentation, add the `V3 N/A` tag.
+- [ ] Link any [Doc Issues](https://github.com/zowe/docs-site/issues) related to this PR.
+- [ ] Fix broken links found in your content areas.
+- [ ] Notify the SME to review this PR (if applicable).
+- [ ] Merge to the `master` branch if your PR updates content that is on the live site. Merge to `docs-staging` if your PR updates content for a future release.
```

Comment:
I'm wondering if this point is necessary or could perhaps cause someone to open a PR against a different branch than that which was used as the base branch (e.g. someone opens a new branch against docs-staging, but then opens to PR against master). Maybe it would be best to say to target the branch that was used to base your PR branch.

Reply:
@janan07 That would be the intent -- to remind folks that the live changes use the master branch and changes for the future use the docs-staging branch. So if their base branch doesn't align w/ this, then, yep, they'd need to change their branch to align w/ our branching strategy.